### PR TITLE
Various science, artist, and blacklist changes

### PR DESCRIPTION
**Pull request by Jamini**
## About The Pull Request

Various small "QoL" and lore tweaks. Mostly around science and artist stuff
- Added most power tools to the science node for power tools
- Fixed pay annoucement text
- Tweaked some art strings to be less lrp memes
- Blacklisted the jenson shades as they are thermals..
- Removed a bad prefix for art
- Removed a lot of references to terran culture that make no sense in our setting
- Chameleon guns now unlock covert tech
- The tool component for armblades and cyberclaws should stop spawning in maint now.

## Why It's Good For The Game

Mostly me doing more small things that people often miss as I catch them.
Note that power tool designs for science still need to be put on a disk before they can be printed as they aren't protolathe designs.

## Changelog
```changelog
tweak: Added most power tools to the science node for power tools
tweak: Removed a bad prefix for art
tweak: Removed a lot of references to terran culture that make no sense in our setting
tweak: Tweaked some art strings to be less lrp memes
fix: The tool component for armblades and cyberclaws should stop spawning in maint now.
fix: Fixed pay annoucement text
balance: Blacklisted the jenson shades as they are thermals..
balance: Chameleon guns now unlock covert tech
```

### CODE REVIEW

File: strings/artist_strings/names/art_weapon_first.txt [hunk 1]
```diff
@@ -80,4 +75,11 @@ Fascist
...
 United
 Federated
-Republican
\ No newline at end of file
+Republican
+Modded
+Transformed
+Arcing
+Synthetic
+Valient
```
Comment (zackerycodestoo) [inline strings/artist_strings/names/art_weapon_first.txt:83]: Grammar police: this should be spelled Valiant

**Review by zackerycodestoo — CHANGES_REQUESTED** [review]
All is well aside from a very small typo

File: strings/artist_strings/names/art_weapon_first.txt (hunk 1 above)
Comment (Jamini) [reply strings/artist_strings/names/art_weapon_first.txt]: Fixed

**Review by zackerycodestoo — APPROVED** [review]
Approved

**Pull request merged**